docs/Guide.md

```diff
@@ -540,6 +541,50 @@ const App = () => {
 
 Note that these cookies will only be sent on the first request unless you use the technique above for [setting custom headers on each page load](#Setting-Custom-Headers).
 
+### Page navigation gesture and button support
+We can provide support for conventional page navigation for both Android and iOS.
+
+For iOS, you'll just need to use the [`allowsbackforwardnavigationgestures`](Reference.md#allowsbackforwardnavigationgestures) prop.
+
+For Android, you can use `useRef` and `useEffect` (you'll need to import them from React if you aren't already) to allow users to navigate to the previous page when they press the back button like so:
```

Comment:
This is only true if you're using functional components and hooks. And I'm totally fine with only providing an example for hooks, I recognize that's the direction things are moving. But I think it would be worth highlighting that the _key_ parts of this solution are in using `BackHandler.addEventListener` and hooking that up to call `goBack` on the WebView.